src/base/projects/Commit/CommitVerifiedBadge.svelte

```diff
@@ -1,14 +1,25 @@
 <script lang="ts">
   import type { CommitMetadata } from "@app/commit";
-  import CommitAuthorship from "./CommitAuthorship.svelte";
+
+  import debounce from "lodash/debounce";
+
   import Badge from "@app/Badge.svelte";
+  import CommitAuthorship from "./CommitAuthorship.svelte";
 
   export let commit: CommitMetadata;
 
-  let hover = false;
+  let visible = false;
+  const showDelay = 50; // ms
```

Comment:
A little sluggish, I'd prefer something closer to 1-2 frames on a 60hz monitor, ie. 16-33ms